### PR TITLE
Add hide online status and hide chat vip image 

### PR DESCRIPTION
**Pull request by NextAlone**
# feat(Purify): add hide online status and hide chat vip image 

## 介绍 / Description

- hide online status
- hide chat vip image 
- fix entry loss of auto renew fire

## 更改内容 / Types of Changes

- 修复 Bugfix
- 新功能 New Feature

## 由这个PR解决或关闭的Issues / Issues Fixed or Closed by This PR

-  

## 检查列表 / Checklist

- [x] 我的代码符合本仓库的代码规范 My code follows the code style of this project
- [x] 我已经测试了这些更改，它们可以正常工作，既不会破坏原有任何功能(或我可以解决)，也不会影响对旧版本的支持 I have tested the changes and verified that they work and don't break anything(as well as I can manage) or drop the support for previous versions.
- [x] 我已合并了对后续工作无意义的commit并确认不会对后续维护造成困扰 I have merged commits that are meaningless for follow-up work and confirmed that they will not cause damage to follow-up maintenance

### CODE REVIEW

**Review by qwq233 — CHANGES_REQUESTED** [review]
拆为3commit 或者我帮你

**Comment by qwq233** [comment]
顺这个feat(Purify)什么意思

File: app/src/main/java/me/nextalone/hook/HideChatVipImage.kt [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+import nil.nadph.qnotified.hook.CommonDelayableHook
+import nil.nadph.qnotified.util.Utils
+
+object HideChatVipImage : CommonDelayableHook("na_hide_chat_vip_image_kt") {
+
+    override fun initOnce(): Boolean {
+        return try {
+            "com.tencent.mobileqq.widget.navbar.NavBarAIO".clazz.hookAfterAllConstructors {
+                val ctx = it.thisObject as RelativeLayout
+                val titleImageId = ctx.resources.getIdentifier("jp0", "id", Utils.PACKAGE_NAME_QQ)
```
Comment (cinit) [inline app/src/main/java/me/nextalone/hook/HideChatVipImage.kt:34]: 你用的 `R.id.jp0` 跨版本兼容性如何?

File: app/src/main/java/me/nextalone/hook/HideOnlineStatus.kt [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+import nil.nadph.qnotified.hook.CommonDelayableHook
+import nil.nadph.qnotified.util.Utils
+
+object HideOnlineStatus : CommonDelayableHook("na_hide_online_status_kt") {
+
+    override fun initOnce(): Boolean {
+        return try {
+            "com.tencent.mobileqq.widget.navbar.NavBarAIO".clazz.hookAfterAllConstructors {
+                val ctx = it.thisObject as RelativeLayout
+                val subTitleLinearLayoutId = ctx.resources.getIdentifier("j65", "id", Utils.PACKAGE_NAME_QQ)
```
Comment (cinit) [inline app/src/main/java/me/nextalone/hook/HideOnlineStatus.kt:35]: RT, 同上

Comment (NextAlone) [reply app/src/main/java/me/nextalone/hook/HideOnlineStatus.kt:35]: at least 8.4.1 up to 8.5.5

File: app/src/main/java/me/nextalone/hook/HideChatVipImage.kt (hunk 1 above)
Comment (NextAlone) [reply app/src/main/java/me/nextalone/hook/HideChatVipImage.kt:34]: at least 8.4.1 up to 8.5.5

Comment (ketal178) [reply app/src/main/java/me/nextalone/hook/HideChatVipImage.kt:34]: at least?
准确点很难吗

Comment (NextAlone) [reply app/src/main/java/me/nextalone/hook/HideChatVipImage.kt:34]: > at least?
> 准确点很难吗
> 

我手机里只有841~855

**Review by qwq233 — APPROVED** [review]

File: app/src/main/java/me/nextalone/hook/HideChatVipImage.kt (hunk 1 above)
Comment (NextAlone) [reply app/src/main/java/me/nextalone/hook/HideChatVipImage.kt:34]: 8.2.0-8.5.5，旧版本id.name不是固定的，是以id命名

**Review by qwq233 — DISMISSED** [review]

**Review by cinit — DISMISSED** [review]

**Review by singleNeuron — DISMISSED** [review]

**Review by qwq233 — APPROVED** [review]

**Review by singleNeuron — APPROVED** [review]

**Review by cinit — APPROVED** [review]

**Pull request merged**